tests/console/gpg.pm

```diff
@@ -172,7 +177,7 @@ sub run {
         libgcrypt20 => '1.9.0',
         'libgcrypt20-hmac' => '1.9.0'
     };
-    zypper_call("in " . join(' ', keys %$pkg_list));
+    zypper_call("in --allow-downgrade " . join(' ', keys %$pkg_list));
```

Comment:
This is extremely dangerous and can cover a lot of issues, mainly with updates.

Reply:
Is `if(check_var('BETA', '1'))` OK for you? Otherwise I'm worried that there is other way for us how to check that.

Reply:
Frankly, I don't think it is a good idea. This can happen when the image is built with a newer package than it is in repo. Thus, repos are not in sync with image. Leading to project issues, and they should be raised by us. Since we are talking about the latest testing, the benefits of this change are questionable.

Reply:
after more thinking about this I  am suggesting the following : 
1. remove `--allow-downgrade` 
2. catch zypper error here and in case `BETA=1` make it soft_fail and skip further test

There are two points why : 
1. As @mloviska correctly mention everywhere except Public Cloud Latest it simply should not be like that 
2. In case of Public Cloud Latest we don't have other choice actually because repos are one step behind the image ( repos for Public Beta contains latest **released** beta while image built on Latest Beta ) BUT if such situation really happens there is no point to run the test anyway because anyway **we won't cover what is on image** because we will downgrade packages so from coverage POV of Latest image there is 0 value of continue this test